Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/SettingsRegistrar.cpp

```diff
@@ -73,7 +73,19 @@ namespace AzToolsFramework
 
         AZ::IO::FixedMaxPath fullSettingsPath = AZ::Utils::GetProjectPath();
         fullSettingsPath /= relativeFilepath;
-        return registry->MergeSettingsFile(fullSettingsPath.Native(), format, anchorKey);
+
+        bool fileExists = AZ::IO::SystemFile::Exists(fullSettingsPath.Native().c_str());
```

Comment:
The AZ::IO::Path class has a `.c_str()` function directly on it and that can be used directly.
```suggestion
        const bool fileExists = AZ::IO::SystemFile::Exists(fullSettingsPath.c_str());
```